supabase/migrations/20230815125300_change_wallet_type.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE wallets ALTER COLUMN wallet_address TYPE text;
+
+ALTER TABLE access DROP CONSTRAINT access_pkey;
+ALTER TABLE access ADD CONSTRAINT access_pkey PRIMARY KEY (user_name, respository);
```

Comment:
primary key was wrong for access table because user can have different access controls for different repos